packages/patternfly-4/react-charts/src/components/ChartDonut/examples/ChartDonut.md

```diff
@@ -20,7 +20,7 @@ In this tutorial, we will build a bar chart together - starting with a simple ch
 a legend, and concluding by changing the theme color. You'll learn how to use React chart components together to build 
 a consistent user experience.
 
-[Start course](https://katacoda.com/patternfly/courses/charts/module-donut)
+[Start course](https://katacoda.com/patternfly/courses/charts/donut-chart)
```

Comment:
When I follow this link, the title for the course is "intro to donut utilization chart components" rather than "donut chart components", but I think it needs to be the latter

<img width="694" alt="Screen Shot 2019-09-19 at 4 25 32 PM" src="https://user-images.githubusercontent.com/32821331/65278459-2a6d5c00-dafa-11e9-9191-7519c6ccabe8.png">

Reply:
Ah, that's a typo in the Katacoda tutorial. I'll have to fix that in another repo. Thanks!

Reply:
Fixed